apps/api/src/app.ts

```diff
@@ -96,6 +98,21 @@ app.all("/mcp", async (c: Context) => {
   }
 });
 
+// Login and auth routes
+Object.entries(loginRoutes).forEach(([route, honoApp]) => {
+  app.route(route, honoApp);
+});
+
+app.all("/api/user", (c: AppContext) => {
```

Comment:
can we use the PROFILES_GET tool call?

Reply:
Sure! I will remove change the UI for this

Reply:
fixed